File: Programming_Basics_with_Python/conditional_statements_advanced _lab/07._working_hours.py
hour_of_day = int(input())
day_of_week = input()
if 10 <= hour_of_day <= 18 and (day_of_week == "Monday" or day_of_week == "Tuesday" or day_of_week == "Wednesday" \
    or day_of_week == "Thursday" or day_of_week == "Friday" or day_of_week == "Saturday"):
    print("open")
else:
    print("closed")

File: Programing_Basics_with_JS/conditionalStatementsAdvancedLab/07.WorkingHours.js
function workingHours(input) {
    let hours = Number(input[0]);
    let day = input[1];

    let openHour = 10;
    let closeHour = 18;
    let openOrClose = 'closed'
    
    if (day === 'Monday' && openHour <= hours && hours <= closeHour) {
        openOrClose = 'open';
    } else if (day === 'Tuesday' && openHour <= hours && hours <= closeHour) {
        openOrClose = 'open';
    } else if (day === 'Wednesday' && openHour <= hours && hours <= closeHour) {
        openOrClose = 'open';
    } else if (day === 'Thursday' && openHour <= hours && hours <= closeHour) {
        openOrClose = 'open';
    } else if (day === 'Friday' && openHour <= hours && hours <= closeHour) {
        openOrClose = 'open';
    } else if (day === 'Saturday' && openHour <= hours && hours <= closeHour) {
        openOrClose = 'open';
    }
    console.log(openOrClose);
}

workingHours(['11', 'Monday'])
workingHours(['19', 'Friday'])
workingHours(['11', 'Sunday'])